spree_admin_roles_and_access.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.2.0'
   s.files = Dir['LICENSE', 'README.md', 'app/**/*', 'config/**/*', 'lib/**/*', 'db/**/*']
 
-  s.author    = ["Nishant 'CyRo' Tuteja", "Akhil Bansal", "Nimish Mehta"]
```

Comment:
@Anuragjain89 Avoid adding authors here. Until it's a version a complete revamp or written from the scratch we don't add contributors to authors.

Reply:
@tanmay3011 

Sure.
I have added a commit `removing my name` from the authors list.